packages/core/src/client/validate.ts

```diff
@@ -103,6 +103,68 @@ export const validateQuery = async (
   }
 };
 
+/**
+ * Find all table names in a SQL query.
+ *
+ * @param sql - SQL query
+ */
+export const findTableNames = async (sql: string) => {
+  // @ts-ignore
+  const Parser = await import(/* webpackIgnore: true */ "pg-query-emscripten");
```

Comment:
I assume the import will be cached, but may make sense to do so explicitly in the file scope?